app/assets/stylesheets/mixins/course/forum/_forum_post.scss

```diff
@@ -0,0 +1,34 @@
+$border_color: #ddd;
```

Comment:
Name of variable `border_color` should be written in all lowercase letters with hyphens instead of underscores